_includes/pbc/all/install-features/202505.0/install-the-order-amendment-feature.md

```diff
@@ -0,0 +1,1296 @@
+| ReplaceSalesOrderThresholdExpensesCheckoutDoSaveOrderPlugin       | Iterates over `QuoteTransfer.expenses` and stores expenses of the type defined by {@link \Spryker\Shared\SalesOrderThreshold\SalesOrderThresholdConfig::THRESHOLD_EXPENSE_TYPE} in the database.        | None                                                                                                                                             | Spryker\Zed\SalesOrderThreshold\Communication\Plugin\Checkout                        |
+| FinishOrderAmendmentCheckoutPostSavePlugin                        | Triggers OMS event defined in {@link \Spryker\Zed\SalesOrderAmendmentOms\SalesOrderAmendmentOmsConfig::getFinishOrderAmendmentEvent()} to finish the order amendment process.                           | None                                                                                                                                             | Spryker\Zed\SalesOrderAmendmentOms\Communication\Plugin\Checkout                     |
+| DummyPaymentCheckoutPostSavePlugin                                | If QuoteTransfer.billingAddress.lastName is 'Invalid' the plugin adds the error into CheckoutResponseTransfer.                                                                                          | None                                                                                                                                             | Spryker\Zed\DummyPayment\Communication\Plugin\Checkout                               |
+| CloseQuoteRequestCheckoutPostSaveHookPlugin                       | If quote contains quote request version reference - marks quote request as closed.                                                                                                                      | None                                                                                                                                             | Spryker\Zed\QuoteRequest\Communication\Plugin\Checkout                               |
+| SendEmailToGiftCardUser                                           | Sends an email to a Gift Card user.                                                                                                                                                                     | None                                                                                                                                             | Spryker\Zed\GiftCardMailConnector\Communication\Plugin\Checkout                      |
```

Comment:
Some of the plugins from this list are from other features and may not exist, for example this one is only for Gift card feature users.